app.js

```diff
@@ -0,0 +1,192 @@
+const playerController = (() => {
+  const Player = function (name, symbol) {
+    this.name = name;
+    this.moves = [];
+    this.symbol = symbol;
+    this.winner = false;
+    this.active = false;
+  };
+  return {
+    addPlayer: function (name, symbol) {
+      return new Player(name, symbol);
+    },
+    addMove: function (move) {
+      this.moves.push(move);
+    },
+
+    addWinener: function () {
+      this.winner = true;
+    },
+
+    activePlayer: function (active) {
+      this.active = active;
+    },
+  };
+})();
+
+const gameBoardController = (function () {
+  const GameBoard = function (xPlayer, oPlayer) {
+    this.xPlayer = xPlayer;
+    this.oPlayer = oPlayer;
+    this.status = "Running";
+    this.currentPlay = xPlayer;
+    this.winner = null;
+    this.playCount = 0;
+  };
+
+  const boardWinner = function (game) {
+    const board = document.getElementsByClassName("box");
+    document.getElementById("game-status").innerHTML = "status = running";
+
+    for (let i = 0; i < board.length; i++) {
+      if (game.playCount < 9) {
+        board[i].addEventListener("click", function () {
+          if (board[i].innerHTML.trim() === "" && game.status === "Running") {
+            board[i].innerHTML = game.currentPlay.symbol;
+            if (game.currentPlay.symbol === "X") {
+              game.currentPlay = game.oPlayer;
+            } else {
+              game.currentPlay = game.xPlayer;
+            }
+            document.getElementById("player").innerHTML = game.currentPlay.name;
+
+            if (
+              board[0].innerHTML === board[1].innerHTML &&
+              board[1].innerHTML === board[2].innerHTML &&
+              board[0].innerHTML.trim() !== ""
+            ) {
+              showWinner(game);
+            } else if (
+              board[3].innerHTML === board[4].innerHTML &&
+              board[4].innerHTML === board[5].innerHTML &&
+              board[3].innerHTML.trim() !== ""
+            ) {
```

Comment:
Let's keep DOM related in a separated file.
![image](https://user-images.githubusercontent.com/31897434/82553144-64946f00-9b5b-11ea-82ef-79089585f49d.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623090/

Reply:
> Let's keep DOM related in a separated file.
> ![image](https://user-images.githubusercontent.com/31897434/82553144-64946f00-9b5b-11ea-82ef-79089585f49d.png)
> https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623090/

## @konyan @gitnyasha 
- Make sure to separate the DOM manipulation tasks from the logic.